engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/aggregates.scala

```diff
@@ -140,24 +140,25 @@ object aggregates {
   }
 
   object LastAggregator extends Aggregator {
-
-    override type Aggregate = AnyRef
+    // TODO: Add test for none.
+    override type Aggregate = Option[AnyRef]
 
     override type Element = AnyRef
 
-    override def zero: Aggregate = null
+    override def zero: Aggregate = None
 
-    override def isNeutralForAccumulator(element: LastAggregator.Element, currentAggregate: LastAggregator.Aggregate): Boolean = false
+    override def isNeutralForAccumulator(element: LastAggregator.Element, currentAggregate: LastAggregator.Aggregate): Boolean =
+      currentAggregate.isDefined
```

Comment:
Is it? For last it's always false, isn't it?

Reply:
It was wrong. I corrected it and added new tests.